tests/regression/26-undefined_behavior/07-arrays-within-structures.c

```diff
@@ -0,0 +1,31 @@
+// PARAM:  --set dbg.debug true --enable ana.arrayoob --enable ana.int.interval --enable ana.int.interval --enable ana.int.enums 
+// Arrays within structures. Source of sample struct:
+// https://codeforwin.org/2018/07/how-to-declare-initialize-and-access-array-of-structure.html
+#include <stdio.h>
+int main() {
+  struct student {
+    char name[100];
+    int roll;
+    float marks;
+  };
+  // Structure array declaration
+  struct student stu[] = {
+      {"vandah", 12, 89.5f},
+      {"edin", 15, 98.0f},
+      {"david", 17, 90.0f},
+  };  
+  // FIXME: this should be OK? but generates an AST error
+  // stu[6].name = "thisisareallylongstringjkdsfjbkslagjbfdghfdnvbfsöajgfklgjfgjlfdsjhhgjgfbldfhghlfdgjöjsadfjhsagjflghfkdsgjhsgblkjsadgfkjabgjlsagdhlshagljhdasghsadgjh"; // WARN
```

Comment:
An assignment of a string to a char array is not possible in this manner, so I would remove the commented-out line.